fiftyone/factory/repos/delegated_operation.py

```diff
@@ -214,6 +214,7 @@ def queue_operation(self, **kwargs: Any) -> DelegatedOperationDocument:
             context.request_params["dataset_id"] = str(op.dataset_id)
             context.request_params["dataset_name"] = context.dataset.name
 
+        op.context = context
```

Comment:
This fixes an existing issue that I think might explain some of the weirdness/handling of `op.context` as `dict`!

Without this change [this](https://github.com/voxel51/fiftyone/pull/5291/files#diff-8c618b37055d71658b7666c9e677d916dfbb8b9f43876a5a2a983be19b63c100R192) is a no-op.